src/Bridge/Laminas/SlugifyService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Cocur\Slugify\Bridge\Laminas;
+
+use Cocur\Slugify\Slugify;
+use Interop\Container\ContainerInterface;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+
+/**
+ * Class SlugifyService
+ * @package    cocur/slugify
+ * @subpackage bridge
+ * @license    http://www.opensource.org/licenses/MIT The MIT License
+ */
+class SlugifyService implements FactoryInterface
```

Comment:
I would suggest a filter here instead of a _service_ because this package provides a conversion / transformation of (string) data.
To create a custom filter please have a look at the official documentation of laminas-filter: "[Writing Filters](https://docs.laminas.dev/laminas-filter/writing-filters/)"